src/schema/nmdc.yaml

```diff
@@ -791,15 +791,18 @@ slots:
 
   eluent_introduction_category:
     range: EluentIntroductionCategoryEnum
-    description: how a processed sample is introduced into a mass spectrometer, for example liquid chromatography or direct infusion through a syringe
+    description: A high-level description for how the processed sample is introduced into a mass spectrometer.
```

Comment:
could this be a "high level categorization"?

Reply:
Yup!! I'll change that in the next PR